ganga/GangaCore/Core/exceptions.py

```diff
@@ -201,3 +201,7 @@ def __init__(self, *args, **kwds):
         super(GangaException, self).__init__(args)
         TypeError.__init__(self, *args)
         self.kwds = kwds
+
+
+class TerminationSignalException(GangaException):
```

Comment:
A `GangaException` is an exception that can occur during the normal operation of `ganga` and will in general be passed back to the user uncaught. So I do *not* think this should be a `GangaException`

Reply:
I see, that makes sense. I will update it to subclass a base exeption in that case